docs/src/components/AppMenu.js

```diff
@@ -8,13 +8,33 @@ import {
   Ripple
 } from 'quasar'
 
-import { mdiArrowDownThinCircleOutline } from '@quasar/extras/mdi-v6'
-import { h, ref, watch, onBeforeUpdate, withDirectives } from 'vue'
+import { mdiMenuDown } from '@quasar/extras/mdi-v6'
+import { h, ref, watch, onBeforeUpdate, withDirectives, computed } from 'vue'
 import { useRoute } from 'vue-router'
+import { Screen } from 'quasar'
+import { navItems } from 'assets/landing-page/nav-items.js'
 
 import Menu from 'assets/menu.js'
 import './AppMenu.sass'
 
+const sidebarMenu = computed(() => {
+  const headerNavPaths = []
+  Object.keys(navItems).forEach(key => {
+    const headerItems = [...navItems[ key ]] // create non referenced copy
```

Comment:
In this case, the code itself is a bit unclear, so a more explicit comment explaining the general need for this computed and why we decided to do it like this would be the best (or rewrite the code to use functions/variables with more clear names)